ClubService.Application/EventHandlers/MemberEventHandlers/MemberDeletedEventHandler.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ClubService.Application.Api;
+using ClubService.Domain.Event;
+using ClubService.Domain.Repository;
+
+namespace ClubService.Application.EventHandlers.MemberEventHandlers;
+
+public class MemberDeletedEventHandler(
+    IMemberReadModelRepository memberReadModelRepository,
+    ITennisClubReadModelRepository tennisClubReadModelRepository,
+    IReadStoreTransactionManager readStoreTransactionManager) : IEventHandler
+{
+    public async Task Handle(DomainEnvelope<IDomainEvent> domainEnvelope)
+    {
+        if (!Supports(domainEnvelope))
+        {
+            return;
+        }
+        
+        var memberReadModel = await memberReadModelRepository.GetMemberById(domainEnvelope.EntityId);
+        
+        if (memberReadModel != null)
```

Comment:
Do reduce nesting we could change the if statement to `if (memberReadModel == null) return;`